README.md

```diff
@@ -1,25 +1,38 @@
-# Forks and Clones
+# Forking and Cloning Github Repositories
+
+## Problem Statement
```

Comment:
I think we should adjust the framing on this (and this is a bit outside the scope of this one lesson). Our previous lesson is on `init; status; commit`. What's the logical move from there to here?

> If you are familiar with how to create _local_ repositories, you have the ability to create a logged history of your project. What's great about open source is that _lots_ of people are creating new repositories all the time and sharing their work. You can take your repository and _push_ it to a _remote_ to share it, or you can _clone_ a repository to your local machine from a _remote_ and make a copy on your own system. Due to permissions issues you might need to _fork_ a repository before you _clone_ it. In this lesson we'll learn the receiving side of this equation: how to _clone_ from a _remote_ to our local system and how to _fork_ when necessary.

I think a PS like that is the logical segue.

Because `fork` and `clone` are so easily confused, I recommend:

1. Define _remote_
2. Use `git clone` to copy a repository to your local machine
3. Use `git remote` to list remotes
4. Use `git fork` via GitHub to clone other organization's repositories into your own.